rcsbapi/search/search_schema.py

```diff
@@ -222,8 +222,10 @@ def _fetch_schema(self, url: str):
 
     def _load_json_schema(self, schema_file):
         logger.info("Loading attribute schema from file")
-        latest = pkgutil.get_data(__package__, schema_file)
-        return json.loads(latest)
+        path = Path(__file__).parent.parent.joinpath(schema_file)
```

Comment:
Can you explain what this does? Why not just use `os.path.abspath(schema_file)`?

Reply:
I found the information online to be confusing, but this is my interpretation: 
This is the recommended method for resources in packages. That is because if a package is installed as a .zip or .whl the resources [don't actually need to be in file system](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files)? In those cases, I don't think `os.path.abspath` would work.

But we haven't packaged our resources in any special way, so I think both would work for us. It looks like this method is generally recommended. 

(And just to be clear, I didn't write this part, so I'm not sure this is why they chose to do it this way)

Reply:
Ah, interesting. Thanks for the info. I guess that might be a better library to use than `os.path`. Something to keep in mind in the future...probably better to be consistent with at least.

Anyway, this looks fine for now, so we can leave as is and merge (I'll go ahead and do that).

Thanks again!